### PR TITLE
Tests for obtaining a service by id

### PR DESCRIPTION
**Pull request by ZdanovichOleg**
1) Created test/services/serviceGetById.spec.js;
2) Modified  helpers/constants.js;
3) Modified  helpers/serviceHelper.js

### CODE REVIEW

File: helpers/constants.js [hunk 1]
```diff
@@ -10,3 +12,11 @@ export const signUpBody = {
...
   password: '123456',
 };
+
+export const vendorId = async () => {
+ return (await vendorHelper.createVendor(vendorHelper.vendorData)).body.payload;
+}
+
+export const serviceId = async () => {
+  return (await serviceHelper.createService(serviceHelper.serviceData(await vendorId()))).body.payload;
+};
```
Comment (RafailFerre) [inline helpers/constants.js:22]: Maybe... is it better if we'll save vendorId and serviceId in vendorHelper ans serviceId?

File: test/services/serviceGetById.spec.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import * as constants from '../../helpers/constants'
+import * as serviceHelper from '../../helpers/serviceHelper'
+import * as vendorHelper from '../../helpers/vendorHelper'
+import {expect} from 'chai'
+const chance = require('chance').Chance();
+describe('A user should get an information about particular service', () => {
+
+    let resService, resServiceWithoutToken, resServiceWithInvalidId;
+    before(async () => {
+        resService = await serviceHelper.getByIdService(await constants.serviceId(), process.env.TOKEN)
```
Comment (RafailFerre) [inline test/services/serviceGetById.spec.js]: Maybe it is better to break separately?

File: test/services/serviceGetById.spec.js [hunk 3]
```diff
@@ -1,67 +1,83 @@
...
+      expect(res.body.message).to.be.a('string');
+    });
+    it('verify type of response payload', () => {
+      expect(res.body.payload).to.be.an('object');
+    });
+    it('verify response status message', () => {
+      expect(res.res.statusMessage).to.eq('OK');
+    });
+    it('verify response payload has keys', () => {
+      expect(res.body.payload).to.have.any.keys(
```
Comment (dumpdusty) [inline test/services/serviceGetById.spec.js]: guess better to use `to.have.all` assertion

File: helpers/serviceHelper.js [hunk 4]
```diff
@@ -17,6 +17,12 @@ export function createService(data) {
     .send(data);
 }
 
+export function getByIdService(serviceId, token) {
```
Comment (dumpdusty) [inline helpers/serviceHelper.js]: maybe `getServiceById` ?

Comment (ZdanovichOleg) [reply helpers/serviceHelper.js]: I have corrected

File: test/services/serviceGetById.spec.js (hunk 3 above)
Comment (ZdanovichOleg) [reply test/services/serviceGetById.spec.js]: I have corrected

**Review by dumpdusty — APPROVED** [review]
thanks for fixing, all looks good

**Review by RafailFerre — APPROVED** [review]

**Pull request merged**